Composer/packages/client/src/components/Split/TopBottomSplit.tsx

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
What's your plan to merge the two splitter codes? (to get rid of the repetitive code in LR and TB splitters)

Reply:
I would do this in the OSS library and then port it here - merging the logic (which I did as a dry run) had a lot of impact on readability.